internal/providers/terraform/ibm/ibm.go

```diff
@@ -57,10 +57,11 @@ var globalCatalogServiceId = map[string]catalogMetadata{
 	"ibm_is_flow_log":               {"is.flow-log-collector", []string{}, nil},
 	"ibm_cloudant":                  {"cloudant", []string{}, nil},
 	"ibm_pi_instance":               {"abd259f0-9990-11e8-acc8-b9f54a8f1661", []string{}, nil},
-	"ibm_cos_bucket":                {"744bfc56-d12c-4866-88d5-dac9139e0e5d:standard-bucket", []string{}, nil},
-	"ibm_is_lb":                     {"is.load-balancer", []string{}, nil},
-	"ibm_is_public_gateway":         {"is.public-gateway", []string{"ibm_is_floating_ip"}, nil},
-	"kms":                           {"ee41347f-b18e-4ca6-bf80-b5467c63f9a6", []string{}, nil},
+	// Huh...
+	"ibm_cos_bucket":        {"dff97f5c-bc5e-4455-b470-411c3edbe49c", []string{}, nil},
```

Comment:
It was decided to use the displayname for the cos bucket from 744bfc56-d12c-4866-88d5-dac9139e0e5d:standard-bucket to differentiate from the object storage instances. This should be reverted back.

Reply:
I know, but it won't be correct with the one-rate plan support. It would point into the wrong object in a different branch of the object tree. I don't see the point of the feature then.
@kzhang-2017

Reply:
@hiltol @balazs-marjan What do we want the display name to be for the one rate support?

Reply:
@balazs-marjan it doesn't really matter to the UI what the underlying object is in global catalog, even if it's pointing to an unrelated plan, as it's only looking for a displayname to use. In this case, the closest thing to a "COS bucket" that is available in global catalog is "Standard Bucket", until something better becomes available. I don't the alternative, which is having everything called a "Cloud Object Storage" is better. The details would then look like below:
<img width="668" alt="Screen Shot 2022-11-07 at 1 10 59 PM" src="https://user-images.githubusercontent.com/101275725/200389065-d074e3ac-31ff-46c0-993f-a59c597c1a09.png">

Reply:
I think the instances themselves should still be called `Cloud Object Storage` but the buckets will be a tougher problem to tackle. I guess we can use `Standard Bucket` for now for the display name...